config/webpack.js

```diff
@@ -0,0 +1,14 @@
+const webpack = require('webpack');
+const MinifyPlugin = require('babel-minify-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'build/ol.js'
+  },
+  devtool: 'source-map',
+  plugins: [
+    new webpack.optimize.ModuleConcatenationPlugin(),
+    new MinifyPlugin()
+  ]
+};
```

Comment:
If not in this branch, we should rename this to be something more specific.  Like `webpack-full-build.js` or something.